DI-1801/convert_counts_to_vcf.py

```
@@ -0,0 +1,305 @@
+            filter=None,
+        )
+        for field in info_fields:
+            field_name = field["column_name"]
+            if field_name in row and pd.notna(row[field_name]):
+                field_type = field["type"]
+                if field_type == "String":
+                    field_value = str(row[field_name])
+                elif field_type == "Integer":
+                    field_value = int(row[field_name])
+
+                record.info[field["id"]] = field_value
```

Pylint (3.3.7)

[error] Possibly using variable 'field_value' before assignment

(E0606)